packages/server/bin/start.ts

```diff
@@ -26,5 +26,19 @@ main(createServer({
     port: 24005,
   },
   testFiles: ["./test/fixtures/*.t.js"],
-  cacheDir: "./tmp/start"
+  cacheDir: "./tmp/start",
+  drivers: {
+    "chrome": {
+      browserName: "chrome",
+      headless: true
+    },
+    "firefox": {
+      browserName: "firefox",
+      headless: true
+    }
+  },
```

Comment:
What is the purpose of having this as configuration, do we actually expect people to change it?

Reply:
So this has given me great consternation as to the best way to represent this. The reason I put the driver config as an opaque piece of information would be so that we could extend it with other drivers in the future. The near future being cloud drivers like BrowserStack. Still, this feels gross, and it should probably be sparse.

The case where I can see people tweaking and changing is when we start adding configurations like adjusted window sizes, network latencies.... minimized vs maximized... all of which Webdriver supports.

Ugly and confusing config is something we want to avoid for sure, so I'd love to figure out ways to make this intuitive.

One thing I've been thinking is that might be easier in the long run if we make our config `.js` instead of `.json`, that way we can make this simpler, as well as making plugins able to provide their configuration options:

```js
const { Local } = require('@bigtest/project');
const { BrowserStack } = require('@bigtest/browserstack');
module.exports = {
  drivers: {
    chrome: Local.chrome(),
    safari: Local.safari(),
    cloudsafari = BrowserStack({
      "platformName": "iOS",
      "deviceName": "iPhone 7",
    })
}
```

Would love to hear your thoughts.

Reply:
Are [these](https://github.com/webdriverio/webdriverio/blob/master/packages/webdriver/webdriver.d.ts#L229) the options that you can configure with Webdriver?

Reply:
As with everything WebDriver it's a bit more complicate :) If I understand correctly, these are the options supported on all WebDrives https://github.com/webdriverio/webdriverio/blob/master/packages/webdriver/webdriver.d.ts#L186-L227

Those others and optionally supported.